core/server/monitor.go

```diff
@@ -300,6 +302,18 @@ func reloadConfig() {
 	proxy.JSSecret = domains.Config.Proxy.Secrets["javascript"]
 	proxy.CaptchaSecret = domains.Config.Proxy.Secrets["captcha"]
 
+	// Check if the Proxy Timeout Config has been set otherwise use default values
+	proxyTimeout := domains.Config.Proxy.Timeout
+	if proxyTimeout.Idle != 0 &&
```

Comment:
These checks require all these to not be 0. However a user might want to only change one variable